doc/content/compiletime.md

```diff
@@ -26,3 +26,19 @@ We're not exactly sure what the nature of the error is, but have
 [an issue](https://github.com/neams-th-coe/cardinal/issues/556)
 to track it. If this is affecting your workflow, please contact us so that
 we can reprioritize it.
+
+
+## Linking Errors After Updating Source
+
+Sometimes, despite following all instructions, `make -j8 MAKEFLAGS=-j8` will not work. For example, if it has been a while since updating Cardinal source, you have cleaned out `build/` and `install/`, and building causes a linking error
+
+```
+Linking libpng: -lpng16 -lz
+Linking Library cardinal/contrib/moose/framework/libmoose-opt.la...
+/usr/bin/ld: cannot find -lptscotchparmetis
+collect2: error: ld returned 1 exit status
+```
+
+Try running `git clean -xfd` from `cardinal/contrib/moose`.
```

Comment:
please add "or wherever you are pointing to MOOSE from with the `MOOSE_DIR` variable"